### DISCUSSION TITLE
Some parameters of official plugins are not in params list

### DISCUSSION THREAD

**Discussion by mr-loop-1** [Green-Software-Foundation/hack#151] [Ask Anything] [answered]
I was looking at the following `e-mem` plugin here https://github.com/Green-Software-Foundation/if-plugins/blob/main/src/lib/e-mem/README.md

and noticed that both the input parameters are not listed in the `params.ts` file of if repo,

**memory/utilization**: percentage of the total available memory being used in the input period
**memory/capacity**: the total amount of memory available, in GB

Are they not canonical or they will be included at a later stage. If I want to include them in my project, should I call them `unofficial` or something?

**Comment by jmcook1186** [comment, marked as answer]
oh, nice catch, thanks. These should be in `params.ts` - raising a ticket to add them now.
